smartsim/_core/_install/configs/mlpackages/LinuxX64CPU.json

```diff
@@ -0,0 +1,56 @@
+{
+    "platform": {
+        "operating_system":"linux",
+        "architecture":"x86_64",
+        "device":"cpu"
+    },
+    "ml_packages": [
+        {
+            "name": "dlpack",
+            "version": "v0.5_RAI",
+            "pip_index": "",
+            "python_packages": [],
+            "lib_source": "https://github.com/RedisAI/dlpack.git"
+        },
+        {
+            "name": "libtorch",
+            "version": "2.4.0",
+            "pip_index": "https://download.pytorch.org/whl/cpu",
+            "python_packages": [
+                "torch<=2.4.0",
+                "torchvision<=0.19.0",
+                "torchaudio<=2.4.0"
+            ],
```

Comment:
If we are going to get these packages from a the extra index, can we suffix them with `+cpu` just to avoid this warning in the build process?

![image](https://github.com/user-attachments/assets/64ffac41-833b-41ec-a60b-1e845c706def)